src/index.js

```diff
@@ -13,15 +13,59 @@ const { wrap } = require('@adobe/openwhisk-action-utils');
 const { logger } = require('@adobe/openwhisk-action-logger');
 const { wrap: status } = require('@adobe/helix-status');
 const { epsagon } = require('@adobe/helix-epsagon');
+const RedirectConfig = require('@adobe/helix-shared/src/RedirectConfig');
```

Comment:
shouldn't we export it properly?

Reply:
depends on https://github.com/adobe/helix-shared/pull/330